### PR TITLE
updating :matches() to :is()

### PR DESCRIPTION
**Pull request by chrisdavidmills**
New PR to update :matches() to :is(), after the previous one failed.

See https://github.com/mdn/browser-compat-data/pull/3092 for the previous context

### CODE REVIEW

File: css/selectors/is.json [hunk 1]
```diff
@@ -99,37 +103,39 @@
...
             "samsunginternet_android": {
               "version_added": false
             },
             "webview_android": {
-              "version_added": true,
-              "alternative_name": ":-webkit-any",
-              "notes": "Doesn't support combinators."
+                "version_added": true,
+                "alternative_name": ":-webkit-any()",
+                "notes": "Doesn't support combinators."
```
Comment (ExE-Boss) [inline css/selectors/is.json]: Wrong indentation.

**Comment by chrisdavidmills** [comment]
Updates made. Thanks @ExE-Boss !

**Comment by chrisdavidmills** [comment]
Fixed

**Comment by Elchi3** [comment]
This PR adds 4 DS_Store files. Please remove them :)

**Comment by chrisdavidmills** [comment]
*sigh*; let's try that again

**Pull request merged**